sdk/n8nsdk/api/openapi-source/v1/handlers/credentials/spec/schemas/update-credential-request.yml

```
@@ -0,0 +1,28 @@
+  name:
+    type: string
+    example: Updated Credential Name
+    description: The name of the credential
+  type:
+    type: string
+    example: githubApi
+    description: The credential type. If changing type, data must also be provided.
+  data:
+    type: object
+    writeOnly: true
+    example: { accessToken: 'new_token_value' }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)